source_code/main_mcu/src/GUI/gui_prompts.c

```diff
@@ -722,17 +722,18 @@ static cust_char_t gui_prompts_get_char_to_display(uint8_t const * current_pin,
     return disp_char;
 }
 
-/*! \fn     gui_prompts_render_pin_enter_screen(uint8_t* current_pin, uint16_t selected_digit, uint16_t stringID, int16_t anim_direction, int16_t vert_anim_direction, int16_t hor_anim_direction, BOOL six_digit_prompt)
+/*! \fn     gui_prompts_render_pin_enter_screen(uint8_t* current_pin, uint16_t selected_digit, uint16_t stringID, int16_t anim_direction, int16_t vert_anim_direction, int16_t hor_anim_direction, BOOL six_digit_prompt, BOOL show_pin)
 *   \brief  Overwrite the digits on the current pin entering screen
 *   \param  current_pin         Array containing the pin
 *   \param  selected_digit      Currently selected digit
 *   \param  stringID            String ID for text query
 *   \param  vert_anim_direction Vertical anim direction (wheel up or down)
 *   \param  hor_anim_direction  Horizontal anim direction (next/previous digit)
 *   \param  six_digit_prompt    TRUE to specify 6 digits prompt
+*   \param  show_pin          TRUE to specify if the PIN digit should be hidden
```

Comment:
isn't it the contrary? maybe provide a more explicit explanation?

Reply:
You're right... The variable was named "hide_pin" and I renamed it but forgot to change the commentary.
I changed and add it more details saying that it is FALSE only when the feature "hide pin" is enabled.